modules/solid_mechanics/include/materials/BackstressRadialReturnStressUpdate.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "RadialReturnStressUpdate.h"
+
+template <bool is_ad>
+class BackstressRadialReturnStressUpdateTempl : public RadialReturnStressUpdateTempl<is_ad>
```

Comment:
I think this class should be called RadialReturnBackStressUpdate.  That is the pattern we used for RadialReturnCreepStressUpdate

Reply:
I had the same thought before, but it would then become RadialReturnBackStressStressUpdate

Reply:
How about RadialReturnBackstressStressUpdate.  I know its redundant but it follows the same pattern.   Thoughts @bwspenc

Reply:
I think I like RadialReturBackstressStressUpdateBase. It's kind of ugly, but it's not exposed to the user so it's not so critical that it's a pretty name.